presto-main/src/main/java/com/facebook/presto/metadata/Metadata.java

```diff
@@ -73,6 +74,11 @@ public interface Metadata
 
     Optional<SystemTable> getSystemTable(Session session, QualifiedObjectName tableName);
 
+    /**
+     * Returns a table handle for time travel expression
+     */
+    Optional<TableHandle> getHandleVersion(Session session, QualifiedObjectName tableName, Optional<Type> tableVersionType, Optional<Block> tableVersionExpression);
```

Comment:
Similar to the other comment. Is there ever a case where `tableVersionType` will be present without `tableVersionExpression` or vice-versa?

Reply:
Fixed